coffee_core/src/coffee.rs

```diff
@@ -349,7 +349,6 @@ impl PluginManager for CoffeeManager {
         let status = repository.upgrade(&self.config.plugins).await?;
         for plugins in status.plugins_effected.iter() {
             self.remove(plugins).await?;
-            // FIXME: pass the verbose flag to the upgrade command
```

Comment:
this commit can be merged inside the commit that you add the verbose field inside the `install`

Reply:
done